rcsbapi/data/schema.py

```diff
@@ -56,6 +58,8 @@ def __init__(self, pdb_url):
         self.node_index_dict = {}
         self.edge_index_dict = {}
         self.type_fields_dict = {}
+        self.name_description_dict = {}
```

Comment:
I think you need to initialize by running `extract_name_description` here because when I initialize, this dict is showing as empty. I think you would also need to switch the order of initializing `name_description_dict` and `seen_names` because `seen_names` must be initialized before running `extract_name_description`

self.seen_names = set()
self.name_description_dict = {}
extract_name_description(args)